catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/rollover/CatalogRolloverAction.java

```diff
@@ -131,7 +131,7 @@ public MetacardImpl doAction(MetacardImpl metacard, File tempFile)
             CreateResponse createResponse = submitStorageCreateRequest(createStorageRequest);
 
             for (Metacard childMetacard : createResponse.getCreatedMetacards()) {
-                LOGGER.info("created catalog content with id={}", childMetacard.getId());
+                LOGGER.trace("created catalog content with id={}", childMetacard.getId());
```

Comment:
Same thought about mapping the created metacards to their ids and logging a single message with the full list.